src/libs/IOUUtils.js

```diff
@@ -1,31 +1,80 @@
 import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import lodashGet from 'lodash/get';
 import CONST from '../CONST';
+import ONYXKEYS from '../ONYXKEYS';
+
+let currencyList = {};
+Onyx.connect({
+    key: ONYXKEYS.CURRENCY_LIST,
+    callback: (val) => {
+        if (_.isEmpty(val)) {
+            return;
+        }
+
+        currencyList = val;
+    },
+});
+
+/**
+ * Returns the number of digits after the decimal separator for a specific currency.
+ * For currencies that have decimal places > 2, floor to 2 instead:
+ * https://github.com/Expensify/App/issues/15878#issuecomment-1496291464
+ *
+ * @param {String} currency - IOU currency
+ * @returns {Number}
+ */
+function getCurrencyDecimals(currency) {
+    const decimals = lodashGet(currencyList, [currency, 'decimals']);
+    return _.isUndefined(decimals) ? 0 : Math.min(decimals, 2);
```

Comment:
If the `currency.decimals` is missing, you should assume that the value is 2, not 0 :)